src/classes/ADDR_Addresses_TDTM.cls

```diff
@@ -1058,24 +1058,9 @@ public class ADDR_Addresses_TDTM extends TDTM_Runnable {
         addr.Geolocation__Latitude__s = (Decimal)sobjSrc.get(strFieldPrefixSrc + 'Latitude');
         addr.Geolocation__Longitude__s = (Decimal)sobjSrc.get(strFieldPrefixSrc + 'Longitude');
 
-        // try to find state and country by code if they aren't specified?
+        // Update the Address' Country and State by Country and State Code if they aren't specified
         if (isStateCountryPicklistsEnabled) {
-
-            // lookup country if needed, by country code
-            String strCountryCode = (String)sobjSrc.get(strFieldPrefixSrc + 'CountryCode');
-            if (null != strCountryCode && null == addr.MailingCountry__c) {
-                addr.MailingCountry__c = validCountriesByCode.get(strCountryCode.toUpperCase());
-            }
-
-            // lookup state if needed, by country code and state code.
-            String strStateCode = (String)sobjSrc.get(strFieldPrefixSrc + 'StateCode');      
-            if (null != strCountryCode && null != strStateCode
-                && (null == addr.MailingState__c
-                    || (null != validStatesByLabel && validStatesByLabel.containsKey(addr.MailingState__c.toUpperCase())
-                        && !strStateCode.equals(validStatesByLabel.get(addr.MailingState__c.toUpperCase()))))) {
-                // assume it's a valid statecode
-                addr.MailingState__c = strStateCode;
-            }
+            updateAddressCountryAndStateWithAccountOrContactCountryCodeAndStateCode(sobjSrc, strFieldPrefixSrc, addr);
```

Comment:
I'm really uncomfortable with the length of method names I'm seeing in this pull request.  I know that @RadSFDeveloper has been promoting using longer names, and I'm willing to be open to change and what the team agrees is best, but this seems too extreme.  With method names this long, I have trouble reading the code and find it harder to grock, than something shorter with a reasonable comment when needed for clarity.  In this specific scenario, I think something like updateEmptyAddressCountryAndState() would have been sufficient.

@njjc @jjbennett @force2b @joy-sfdo @RadSFDeveloper thoughts?

Reply:
@davidhabib -- Sounds good. The end goal in naming is clarity. I think striving to be descriptive and succinct is a great way to move forward.